### PR TITLE
[Interface] invalid param case

### PR DESCRIPTION
**Pull request by jaeyun-jung**
1. Add condition to check invalid param of agent interface.
2. Code clean, fix param name and indent.

### CODE REVIEW

File: daemon/include/ml-agent-interface.h [hunk 1]
```diff
@@ -119,29 +119,29 @@ gint ml_agent_model_activate(const gchar *name, const guint version, GError **er
  * @brief An interface exported for getting the description of the given model of @name and @version
  * @param[in] name A name indicating the model whose description would be get
  * @param[in] version A version of the given model, @name
- * @param[out] description Return location for the description of the given model of @name and @version
+ * @param[out] model_info Return location for the information of the given model of @name and @version
```
Comment (myungjoo) [inline daemon/include/ml-agent-interface.h:122]: 1. Please specify whether the caller should free ```model_info``` or not.
2. The location is the path to the model? (the current description is a bit ambiguous)

Comment (jaeyun-jung) [reply daemon/include/ml-agent-interface.h:122]: I updated API description,
- add remarks to release returned string.
- location > pointer

**Comment by myungjoo** [comment]
uh.. wait.. we tried to avoid using GLIB in ML APIs!

**Review by myungjoo — APPROVED** [review]
LGTM.

However, pleasey remove glib-dependency in API interface in later PRs.

**Comment by jaeyun-jung** [comment]
> LGTM.
> 
> However, pleasey remove glib-dependency in API interface in later PRs.

Wook will revise all interfaces of ML agent.

**Review by anyj0527 — APPROVED** [review]
LGTM

**Review by gichan-jang — APPROVED** [review]
LGTM

**Pull request merged**